### ISSUE TITLE
Some required packages have custom error messages, other's don't

### ISSUE THREAD

**Issue by pmer** [Driftwood2D/Driftwood#197]
Compare these two:

```
$ ./driftwood             
Driftwood 2D
Starting up...
[0] FATAL: jsonschema required, module "jsonschema" not found
```

```
$ ./driftwood                 
Traceback (most recent call last):
  File "/usr/lib64/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib64/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "./driftwood/__main__.py", line 64, in <module>
  File "./driftwood/driftwood.py", line 41, in <module>
  File "./driftwood/databasemanager.py", line 31, in <module>
ModuleNotFoundError: No module named 'ubjson'
```

**Comment by pmer** [comment]
We should pick one approach and stick to it.

If we want the former, maybe we could make a `list()` of packages we require and we try to import them at startup?

**Comment by seisatsu** [comment]
This was an oversight. I'll add checks for ubjson at startup.

**Comment by seisatsu** [comment]
Fixed in 9f3c166.
